mix.exs

```diff
@@ -5,6 +5,7 @@ defmodule Events.Mixfile do
     [app: :events,
      version: "0.1.0",
      elixir: "~> 1.4",
+     escript: [main_module: CommandParser],
```

Comment:
@nicolas-simplex is this necessary? I think this might be part of the issue you're working on.